### PR TITLE
add code coverage

### PR DESCRIPTION
**Pull request by Johann-S**
We this PR we will be sure our JS is covered at 80% but we are free to grow this percentage 😉 

BTW it should throw an error if we move under 80% of covered code

Fixes : #25748 

/CC @XhmikosR, @bardiharborow

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,6 +39,7 @@
     "js-compile-standalone": "rollup --environment BUNDLE:false --config build/rollup.config.js --sourcemap",
     "js-compile-bundle": "rollup --environment BUNDLE:true --config build/rollup.config.js --sourcemap",
     "js-compile-plugins": "cross-env PLUGINS=true babel js/src/ --out-dir js/dist/ --source-maps",
+    "js-compile-plugins-coverage": "cross-env PLUGINS=true NODE_ENV=test babel js/src/ --out-dir js/dist/coverage/ --source-maps",
```
Comment (Johann-S) [inline package.json]: maybe I should change the output dir to : `js/coverage/dist` which is not distributed through npm 🤔

**Comment by Johann-S** [comment]
it's give us something like this in `js/coverage/lcov-report/index.html`

image

**Comment by XhmikosR** [comment]
Shouldn't we have a service like Travis so that we can check quickly things? I'm pretty sure there are various services we could integrate.

**Comment by Johann-S** [comment]
Travis execute it for us see : https://travis-ci.org/twbs/bootstrap/jobs/348736259#L704

**Comment by XhmikosR** [comment]
OK, I'm just used to coverage as a repo hook. But this works too.

**Comment by Johann-S** [comment]
I thinks it's a good first step and it's pretty convenient to be able to run the coverage locally , but I'm not against a hook in an other PR

**Comment by Johann-S** [comment]
Would you like that PR @XhmikosR ? Or do you have any concerns ? If not maybe can you approve my PR 😄

**Review by XhmikosR — APPROVED** [review]

**Comment by XhmikosR** [comment]
@Johann-S; just make sure you resolve the conflicts right :)

**Pull request merged**